### PR TITLE
Fix upload test-helper

### LINKED ISSUE #144: upload test-helper does not trigger upload
Hi, I can't figure out what am I doing wrong, but the upload helper is not working for me. I have a following acceptance test

```js
module('Acceptance | upload', function(hooks) {
  setupApplicationTest(hooks);

  test('upload works', async function(assert) {
    await visit('/');

    let file = File.fromDataURL('data:image/gif;base64,R0lGODdhCgAKAIAAAAEBAf///ywAAAAACgAKAAACEoyPBhp7vlySqVVFL8oWg89VBQA7');
    await upload('#upload-photo', file, 'smile.gif');

    let photo = server.db.photos[0];
    assert.equal(photo.filename, 'smile.gif');
  });
});
```

which is basically what's in the README. I can see that upload helper is being called with my input selector, but when I put a `debugger;` into the JS code, I can also see that my `onaddfile` handler is not called and thus nothing is uploaded to the server.

What am I doing wrong?

EDIT:

After a bit of investigation, I've found this line https://github.com/tim-evans/ember-file-upload/blob/latest/addon/queue.js#L68. The problem here is that in the example we use File (ember model) but this code requires File (native browser object).

I also had to comment out this line https://github.com/tim-evans/ember-file-upload/blob/latest/test-support/helpers/upload.js#L6 because File.name is readonly. 

Now it's working. 

I can do a PR to fix this issue, if you're interested.

### PR DESCRIPTION
**Pull request by ondrejsevcik**
This should fix #144, #138 and #70 
Inspired by #89

### CODE REVIEW

File: test-support/helpers/upload.js [hunk 1]
```diff
@@ -1,19 +0,0 @@
-/*global triggerEvent, find, findWithAssert */
```
Comment (ondrejsevcik) [inline test-support/helpers/upload.js]: I've moved this one into `addon-test-helpers` to allow for easy import

`import {upload} 'ember-file-upload/test-support'`

We probably don't need to keep backward compatibility in here as didn't work anyway.

Comment (tim-evans) [reply test-support/helpers/upload.js]: I think this worked for prior versions of ember (pre 3.0)

Comment (tim-evans) [reply test-support/helpers/upload.js]: Could you keep this, just in case? I know it may be broken, but I'd prefer not to major version bump.

Comment (ondrejsevcik) [reply test-support/helpers/upload.js]: It's back.

**Comment by tim-evans** [comment]
:heart: I’ll get this in and released soon!

**Pull request merged**